compiler/luci/pass/src/FuseInstanceNormPass.cpp

```diff
@@ -381,13 +551,28 @@ namespace luci
 bool FuseInstanceNormPass::run(loco::Graph *g)
 {
   bool changed = false;
+  luci::CircleAdd *add;
+  InstanceNormPattern::PatternVersion pv;
+
   for (auto node : loco::active_nodes(loco::output_nodes(g)))
   {
-    auto add = dynamic_cast<luci::CircleAdd *>(node);
-    if (not add)
-      continue;
+    auto reshape = dynamic_cast<luci::CircleReshape *>(node);
```

Comment:
(Optional) You could just check the opcode.

Reply:
`reshape` is used on the line 569, so we need this variable anyway. I think we can leave it as-is for now.